README.md

```diff
@@ -6,8 +6,12 @@ Welcome to a custom **Copilot** workshop prepared by the **GitHub’s Expert Ser
 
 This workshop involves several technology stacks, therefore it is essential that you need to meet prerequisites as outlined in the prerequisites getting started if you want to follow along. Otherwise, please follow along and listen to the GitHub Expert Service team's instructor demo to see how it works in practice.
 
-> :blue_book: **NOTE**: This workshop will be conducted through **Codespace**, but you can find a full step-by-step self-paced guide that you can try later without Codespace:
-[**gh.io/octofit-tutorial**](https://gh.io/octofit-tutorial)
+If you want to checkout this tutorial later after the workshop, please click on the links below:
+
+[gh.io/octofit-tutorial-codespace](https://gh.io/octofit-tutorial-codespace)
```

Comment:
I still don't think we need a separate tutorial and this ponts to the short link that already exists in our discussion [https://github.com/orgs/community/discussions/142536.](https://github.com/orgs/community/discussions/142536.) I feel like what we have is step by step. ex. the guy that tried to help us got to the same end result for the Django REST framework

Reply:
Even if we are doing a demo in the Codespace, it is highly likely there will be some users who cannot finish the demo using Codespace (and they might want to continue doing that way). However, for CORS issue and other things, some stuffs will be easier to work without Codespace, and some users might want to try that, hence why I think it is not a bad idea to provide a guide without Codespace. And I think redundancy does not necessarily hurt in this case but help.